assets/check

```diff
@@ -72,8 +77,11 @@ fi
 
 if [ -n "$tag_filter" ]; then
   {
-    if [ -n "$ref" ]; then
+    if [ -n "$ref" ] && [ -n "$branch" ]; then
       git tag --list "$tag_filter" --sort=creatordate --contains $ref
+    elif [ -n "$ref" ]; then
+      ref_for_search=$(sed 's#/#\\/#g' <<< $ref)
+      git tag --list "$tag_filter" --list --sort=creatordate | sed -ne "/$ref_for_search/,$ p"
```

Comment:
My `sed`-fu leaves much to be desired, would you mind adding a comment (either here or in-code) explaining these two lines or refactoring this into a function describing what it does? Also curious about the extra `--list` flag.

If I had to guess, it looks like this avoids the use of `--contains` (which is what originally  and instead just lists the tags in order and trims all the tags preceding the given ref.

Reply:
yea definitely need to make this clearer! Essentially the first does a replacement on escape characters in $ref so the second line can search and only outputs tags after $ref. That second --list is baffling me it must have been a typo! it doesn't seem to harm things. I have also realised there are a few more characters I need to escape for the first sed so will address these things tomorrow when I update the pr.